### PR TITLE
Snippet Description Edits

### PR DESCRIPTION
**Pull request by nicksieger**
Copy-editing pass on snippet title and descriptions. I think the snippets communicate best when the titles and descriptions have the template "IMPERATIVE-VERB a NOUN-PHRASE", where the imperative verb is not using a passive voice or a weak verb like "Use".

- snippets: edit descriptions for brevity
- snippets: new snippet for injecting image into existing k8s resource

### CODE REVIEW

File: src/_data/snippets/k8s_configmap.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Create a K8s configmap
-description: Create a new configmap with a Tilt extension
+description: Use a Tilt extension to create a configmap
```
Comment (nicksieger) [inline src/_data/snippets/k8s_configmap.yml:3]: Argh. This is just the title reversed. Looking for better options here

File: src/_data/snippets/k8s_secret.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Create a K8s secret
-description: Create a new K8s secret with a Tilt extension
+description: Use a Tilt extension to create a K8s secret
```
Comment (nicksieger) [inline src/_data/snippets/k8s_secret.yml:3]: Same redundancy with the title here

File: src/_data/snippets/min_tilt_version.yml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
 title: Enforce a minimum Tilt version
-description: |
-  Require a minimum version of Tilt to ensure all expected features are available.
+description: Require a minimum version of Tilt for feature availability
```
Comment (nicksieger) [inline src/_data/snippets/min_tilt_version.yml:3]: I was also wondering whether this snippet should just use
```
version_settings(check_updates=True, constraint='>=0.23.7')
```

Comment (milas) [reply src/_data/snippets/min_tilt_version.yml:3]: Oh yeah, that's way better! I forgot about that built-in 😬

File: src/_data/snippets/docker_build_simple.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Basic docker_build
-description: Build a docker image
+title: Build a docker image
```
Comment (milas) [inline src/_data/snippets/docker_build_simple.yml:2]:
```suggestion
title: Build a Docker image
```

**Review by milas — APPROVED** [review]
Nice clean up!

**Review by hyu — APPROVED** [review]
This is excellent! I love the consistency here, @nicksieger. It makes it more predictable and faster to scan these snippets to look for the right one.

**Pull request merged**